employee-rostering-benchmark/src/main/resources/employeeRosteringBenchmarkConfig.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plannerBenchmark>
+  <benchmarkDirectory>local/benchmarkReport</benchmarkDirectory>
+  <warmUpSecondsSpentLimit>30</warmUpSecondsSpentLimit>
+
+  <inheritedSolverBenchmark>
+    <solver>
+      <solutionClass>org.optaweb.employeerostering.domain.roster.Roster</solutionClass>
+      <entityClass>org.optaweb.employeerostering.domain.shift.Shift</entityClass>
+
+      <scoreDirectorFactory>
+        <scoreDrl>org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl</scoreDrl>
+      </scoreDirectorFactory>
+
+      <termination>
+        <secondsSpentLimit>10</secondsSpentLimit>
+      </termination>
+    </solver>
+  </inheritedSolverBenchmark>
+
+  <solverBenchmark>
+    <name>Default</name>
+    <solver>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Move Selector</name>
+    <solver>
+      <localSearch>
+        <unionMoveSelector>
+          <changeMoveSelector/>
+          <swapMoveSelector/>
+        </unionMoveSelector>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Pillar Change Move Selector</name>
+    <solver>
+      <localSearch>
+        <unionMoveSelector>
+          <pillarChangeMoveSelector/>
+        </unionMoveSelector>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Pillar Swap Move Selector</name>
+    <solver>
+      <localSearch>
+        <unionMoveSelector>
+          <pillarSwapMoveSelector/>
+        </unionMoveSelector>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Pillar Change and Swap Move Selector</name>
+    <solver>
+      <localSearch>
+        <unionMoveSelector>
+          <pillarChangeMoveSelector/>
+          <pillarSwapMoveSelector/>
+        </unionMoveSelector>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
```

Comment:
In the interest of brevity, I'd remove these. Three benchmarks (without pillars, with and with sequential) should be enough to demonstrate the functionality.